plugins/rtmp-services/data/services.json

```diff
@@ -1868,17 +1868,12 @@
                 }
             ],
             "recommended": {
-                "keyint": 2,
+                "keyint": 1,
                 "profile": "high",
-                "supported resolutions": [
-                    "1920x1080",
-                    "1280x720",
-                    "960x540",
-                    "640x360"
-                ],
                 "max fps": 60,
-                "max video bitrate": 6000,
+                "max video bitrate": 10000,
                 "max audio bitrate": 192,
+                "bframes": 0,
```

Comment:
`tune=zerolatency` should already be setting bframes to 0, so this seems redundant.

Reply:
Actually, I suppose this setting would be used on hardware encoders? I not certain what OBS does with x264opts when you're not using x264.

Reply:
Yes, I added `bframes: 0` for hardware encoders (but haven't tested it yet)

Reply:
I don't really think it's necessary even for hardware encoders. Hardware encoders already have very low latency. I'd feel more comfortable without this.

Reply:
removed